server/config.js

```diff
@@ -81,6 +87,12 @@ var conf = convict({
         doc: 'Require SSL for postgres connections.',
         default: false,
         format: Boolean
+      },
+      queryTimeout: {
+        doc: 'Milliseconds before app-level queries will bail',
+        default: 15000,
+        env: 'PGTIMEOUT',
```

Comment:
`ES_TIMEOUT` vs `PGTIMEOUT`. Not sure if we should add an underscore or PG or remove it on ES.